wtxmgr/db.go

```diff
@@ -241,35 +257,63 @@ func keyBlockRecord(height int32) []byte {
 }
 
 func valueBlockRecordEmpty(block *BlockMeta) []byte {
-	v := make([]byte, 46)
+	v := make([]byte, 47)
 	copy(v, block.Hash[:])
 	byteOrder.PutUint64(v[32:40], uint64(block.Time.Unix()))
 	byteOrder.PutUint16(v[40:42], block.VoteBits)
-	byteOrder.PutUint32(v[42:46], 0)
+	byteOrder.PutUint32(v[43:47], 0)
```

Comment:
The diff definitely did not show those comments so they looked very much like magic.